README.md

```diff
@@ -286,11 +291,31 @@ spec:
     image: your-customized-auto-instrumentation-image:python
   dotnet:
     image: your-customized-auto-instrumentation-image:dotnet
+  apache:
+    image: your-customized-auto-instrumentation-image:apache
 ```
 
 The Dockerfiles for auto-instrumentation can be found in [autoinstrumentation directory](./autoinstrumentation).
 Follow the instructions in the Dockerfiles on how to build a custom container image.
 
+>**Note:** For `Apache` auto-instrumentation, by default, instrumentation assumes httpd version 2.4. If you need to use version 2.2, or you need to adjust agent parameters, adjust instrumentation per following example:
```

Comment:
This deserves a new section

Reply:
Hello Pavol,

thank you for your review. Based on you comments, I'd do a new set of PR's
1) auto-instrumentation image build and documentation
2) changes related to Instrumentation CRD - definition, validation, etc.
3) Injection itself + e2e tests
4) default settings in the operator

Would that be OK?

Reply:
yes, that would be perfect ❤️